sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/aggregator/Aggregator.java

```diff
@@ -50,8 +50,10 @@
    * instruments to create {@code Accumulation} that are passed to the processor.
    *
    * @param value the given value to be used to create the {@code Accumulation}.
-   * @return a new {@code Accumulation} for the given value.
+   * @return a new {@code Accumulation} for the given value, or {@code null} if there are no
```

Comment:
I'm assuming there's no "empty value" we can use instead?

Reply:
This shouldn't ever be returning `null` AFAIK.  I believe the only use case for `null` here is the `NoAggregator` / default.

Reply:
I'm a fan of either or both of those.  Right now EmptyAggregator shouldn't have any of its methods called in practice, it's just used as a placeholder to disable metrics via views, the storage should never actually be wired up with it, so all of this gymnastics is just to appease our tooling since we can't express that aspect simply.

TL;DR; I'm a fan of Jack's proposal